### PR TITLE
Closes #32: class SoftmaxRegressionModel added + small changes in RNNModel

### LINKED ISSUE #32: Implement simple Logistic Regression model

### PR DESCRIPTION
**Pull request by michaljaworski259**

### CODE REVIEW

**Review by jkoscialkowski — CHANGES_REQUESTED** [review]
Remove F.softmax, reshape input tensor X

**Review by jkoscialkowski — APPROVED** [review]
